### PR TITLE
Display study description in series overview

### PR DESCRIPTION
**Pull request by mollybuckley**
The study description from DICOM headers (0008,1030) is now displayed in the table on the series overview page.

Screenshot 2024-02-08 at 14 15 37

### CODE REVIEW

**Comment by sophie22** [comment]
Could you change this table such that it includes all of the information in this order:
- Acquisition timestamp (year, month, day only)
- Study description
- Manufacturer
- Device name
- Series description
- Files in Series
- Uploaded date
- Has results (rename the current row of 'Reports', can keep the same value)

**Comment by mollybuckley** [comment]
This is now ready for review again - the table in series view looks like this:

image

File: app/main/routes.py [hunk 1]
```diff
@@ -256,6 +256,10 @@ def series_view(series_id):
     user_id = current_user.id
     # Retrieve the Series that was selected
     series = Series.query.filter_by(id=series_id).first_or_404()
+    study_id = series.study_id
+    device_id=series.device_id
+    study = Study.query.filter_by(id=study_id).one_or_none()
+    device = Device.query.filter_by(id=device_id).first()
```
Comment (sophie22) [inline app/main/routes.py:262]: why did you choose different functions for these?
above there is use of `first_or_404` and then `one_or_none` and `first`. I'm wondering what the difference is between these, and what's the rationale for using one over the other.

**Review by sophie22 — APPROVED** [review]
This looks great, thanks for the changes Molly!


I added a tiny comment mostly for educational purposes.

**Pull request merged**